packages/node/test/helper/settimeouttransport.ts

```diff
@@ -0,0 +1,14 @@
+import { SentryEvent, SentryResponse, Status } from '../../src';
+import { BaseTransport } from '../../src/transports';
+
+export class SetTimeoutTransport extends BaseTransport {
+  public async captureEvent(_: SentryEvent): Promise<SentryResponse> {
+    return new Promise<SentryResponse>(resolve => {
+      setTimeout(() => {
+        resolve({
+          status: Status.fromHttpCode(200),
+        });
+      }, (process as any).SENTRY_REQUEST_TIMEOUT || 1);
```

Comment:
We don't use `SENTRY_REQUEST_TIMEOUT` anywhere, so not sure if this is necessary